outofcore/include/pcl/outofcore/octree_base_node.h

```diff
@@ -477,9 +477,8 @@ namespace pcl
         void
         loadFromFile (const boost::filesystem::path &path, OutofcoreOctreeBaseNode* super);
 
-        /** \brief Recursively converts data files to ascii XZY files
-         *  \note This will be deprecated soon
-         */
+        /** \brief Recursively converts data files to ascii XZY files */
+        PCL_DEPRECATED(1, 12, "")
```

Comment:
```suggestion
        PCL_DEPRECATED(1, 13, "")
```

Since it says: will be deprecated